### ISSUE TITLE
Nested package names

### ISSUE THREAD

**Issue by nunofgs** [bojand/grpc-inspect#10]
First of all, thank you for the great lib :)

I'm having issues with a package name that includes dots, such as:

```js
package foo.bar;

service FooService {
  rpc DoSomething() returns () {}
}
```

When fetching the service names (`grpcInspect(proto).serviceNames()`) it returns an empty array.

Is this a bug?

**Comment by codesword** [comment]
I am also having issues with this.

**Comment by bojand** [comment]
Hello, this was actually addressed in 0.5.0. Here is the [test proto](https://github.com/bojand/grpc-inspect/blob/master/test/protos/dotpkg.proto) as well as the actual [test](https://github.com/bojand/grpc-inspect/blob/master/test/grpcinspect.test.js#L246) I forgot to reference it in this issue and close it. Please let me know if it's still an issue.
